tsconfig.base.json

```diff
@@ -144,6 +144,9 @@
         "libs/components/split-view/testing/src/public-api.ts"
       ],
       "@skyux/storybook": ["libs/components/storybook/src/index.ts"],
+      "@skyux/storybook/font-loading": [
+        "libs/components/storybook/src/lib/font-loading/font-loading.service.ts"
+      ],
```

Comment:
Importing from `@skyux/storybook` caused side effects from `@storybook/core`. This allows the `integration` project to import one service without the side effects.